elastic/check.py

```diff
@@ -356,8 +356,11 @@ def get_instance_config(self, instance):
         # Support URLs that have a path in them from the config, for
         # backwards-compatibility.
         parsed = urlparse.urlparse(url)
-        if parsed[2] != "":
+        if parsed[2] != "" and parsed[2] == '/':
```

Comment:
This seems to be in contradiction with the comments above. Also it make this if/else useless since we now always leave the path in the URL (we only remove arguments).

Also since `url` is used in the tags line 376 it may break dashboard and monitors for customers.